lib/history.js

```diff
@@ -1,12 +0,0 @@
-import Ably from "ably/promises";
```

Comment:
deleted because if you are connecting to a channel and need to get its history, you should use rewind to prevent race conditions where current messages arrive on the channel as history is being retrieved, which could lead to message ordering issues during render.